go.mod

```diff
@@ -9,4 +9,5 @@ require (
 	github.com/pkg/errors v0.8.1 // indirect
 	github.com/stretchr/testify v1.3.0
 	github.com/zondax/hid v0.9.0 // indirect
+	github.com/zondax/ledger-go v0.9.0
```

Comment:
Since this was changed, the old dependency should probably be removed from `go.mod` and `go.sum`.